packages/store/src/tables/Callbacks.sol

```diff
@@ -47,38 +49,34 @@ library Callbacks {
   /** Get value */
   function get(bytes32 key) internal view returns (bytes24[] memory value) {
     bytes32[] memory _primaryKeys = new bytes32[](1);
-
-    _primaryKeys[0] = bytes32(key);
+    _primaryKeys[0] = bytes32((key));
```

Comment:
just carrying over discord convo: these extra parens are because our autogen optionally wraps values and, to keep autogen simpler (less inline conditionals), we just leave the parens here.